### PR TITLE
Furniture/Terrain Description Proofreading

### PR DESCRIPTION
**Pull request by FoolsGold45**
#### Summary
None

#### Purpose of change
Fixing grammar mistakes, punctuation, and awkward phrasing in descriptions of furniture and terrain. Standardizing many of the description formats for consistency's sake. Occasionally adding some new content for flavor and/or player utility. To enhance readability and user understanding of what the various world objects are.

#### Testing

The only thing changed are json descriptions, nothing else should be affected by this.

#### Additional context

I'm happy to revert any changes deemed unnecessary.

### CODE REVIEW

**Review by Knut-Aage-Hofseth — APPROVED** [review]

File: data/json/furniture_and_terrain/terrain-flora.json [hunk 1]
```diff
@@ -2590,17 +2536,5 @@
       "ter_set": "t_dirt",
       "items": [ { "item": "withered", "prob": 50, "count": [ 1, 2 ] }, { "item": "leaves", "count": [ 1, 2 ] } ]
     }
-  },
-  {
-    "type": "terrain",
```
Comment (Maleclypse) [inline data/json/furniture_and_terrain/terrain-flora.json:2595]: Did you intend to remove the fungus colony terrain?

**Review by Maleclypse — COMMENTED** [review]
You have a number of non grammar changes that I think mean  you needed to rebase on the most recent master to fix.

File: data/json/furniture_and_terrain/terrain-flora.json [hunk 2]
```diff
@@ -1564,60 +1564,6 @@
       "items": [ { "item": "stick_long", "count": [ 3, 10 ] }, { "item": "splinter", "count": [ 10, 25 ] } ]
     }
   },
-  {
-    "type": "terrain",
```
[the remark this answers is not in the document]
Comment (FoolsGold45) [reply data/json/furniture_and_terrain/terrain-flora.json:1568]: This edit and all the others below were not intended. The repository I'm working off of is currently only 42 commits behind the Clever Raven master, and none of those commits seem to have anything to do with these files at first glance so I have no idea how this happened.

[the remark this answers is not in the document]
Comment (FoolsGold45) [reply data/json/furniture_and_terrain/terrain-flora.json:1568]: I do not know why this is a change from the master. The repository I made changes in for this PR is currently only 42 commits out-of date with the Clever Raven master and those 42 do not seem to interact with these files, so I'm really confused about how this happened. Going to see about rebasing.

**Comment by FoolsGold45** [comment]
> Could you copy over the description changes to furniture in item form as well? They used the same exact descriptions so both should be edited.

#62503 has been in limbo for a while and reflects my edits to the item forms. I will do my best to match the descriptions for when both get merged.

File: data/json/furniture_and_terrain/terrain-flora.json [hunk 3]
```diff
@@ -1566,6 +1566,7 @@
   },
   {
     "type": "terrain",
+Maleclypse marked this conversation as resolved.
```
Comment (Maleclypse) [inline data/json/furniture_and_terrain/terrain-flora.json:1569]:
```suggestion
```

File: data/json/furniture_and_terrain/terrain-flora.json [hunk 4]
```diff
@@ -2602,5 +2603,6 @@
     "coverage": 80,
     "flags": [ "TRANSPARENT", "FLAMMABLE_ASH", "NOCOLLIDE" ],
     "bash": { "str_min": 2, "str_max": 6, "ter_set": "t_dirt", "sound": "crunch.", "sound_fail": "whish." }
+    }
   }
```
Comment (Maleclypse) [inline data/json/furniture_and_terrain/terrain-flora.json:2607]:
```suggestion
```

**Comment by Maleclypse** [comment]
Ok I went ahead and manually fixed all the weird syncing errors I found. There were a lot which is not your fault but there's a good chance I introduced a new error somewhere in the mix.  Once it's done testing if there are any issues please feel free to ping me on discord and I'll come see if I can fix them.

File: data/json/furniture_and_terrain/terrain-flora.json [hunk 5]
```diff
@@ -2589,7 +2589,7 @@
       "sound_fail": "whish.",
       "ter_set": "t_dirt",
       "items": [ { "item": "withered", "prob": 50, "count": [ 1, 2 ] }, { "item": "leaves", "count": [ 1, 2 ] } ]
-    }
+      },
```
Comment (Maleclypse) [inline data/json/furniture_and_terrain/terrain-flora.json:2592]:
```suggestion
      }
```

Comment (Maleclypse) [inline data/json/furniture_and_terrain/terrain-flora.json:2592]:
```suggestion
```

File: data/json/furniture_and_terrain/terrain-flora.json [hunk 6]
```diff
@@ -2589,7 +2589,6 @@
       "sound_fail": "whish.",
       "ter_set": "t_dirt",
       "items": [ { "item": "withered", "prob": 50, "count": [ 1, 2 ] }, { "item": "leaves", "count": [ 1, 2 ] } ]
```
Comment (Maleclypse) [inline data/json/furniture_and_terrain/terrain-flora.json:2591]:
```suggestion
      "items": [ { "item": "withered", "prob": 50, "count": [ 1, 2 ] }, { "item": "leaves", "count": [ 1, 2 ] } ]
      }
```

File: data/json/furniture_and_terrain/appliances.json [hunk 7]
```diff
@@ -549,7 +549,7 @@
     "type": "vehicle_part",
     "name": { "str": "makeshift ammonia machine pipework" },
     "item": "ammonia_machine_pipework",
-    "description": "A rather complicated system of thick-walled pipes, powerful compressors and wiring meant to interface with a reactor producing ammonia from nitrogen and hydrogen via Haber-Bosch process, with a power cord to connect it to an electric grid.",
+    "description": "A rather complicated system of thick-walled pipes, powerful compressors, and wiring, meant to interface with a reactor producing ammonia from nitrogen and hydrogen via the Haber-Bosch process, with a power cord to connect it to an electric grid.",
```
Comment (Maleclypse) [inline data/json/furniture_and_terrain/appliances.json:552]:
```suggestion
    "description": "A rather complicated system of thick-walled pipes, powerful compressors, and wiring, meant to interface with a reactor producing ammonia from nitrogen and hydrogen via the Haber-Bosch process, with a power cord to connect it to anelectric grid.",
```

File: data/json/furniture_and_terrain/appliances.json [hunk 8]
```diff
@@ -549,7 +549,7 @@
     "type": "vehicle_part",
     "name": { "str": "makeshift ammonia machine pipework" },
     "item": "ammonia_machine_pipework",
-    "description": "A rather complicated system of thick-walled pipes, powerful compressors and wiring meant to interface with a reactor producing ammonia from nitrogen and hydrogen via Haber-Bosch process, with a power cord to connect it to an electric grid.",
+    "description": "A rather complicated system of thick-walled pipes, powerful compressors, and wiring, meant to interface with a reactor producing ammonia from nitrogen and hydrogen via the Haber-Bosch process, with a power cord to connect it to anelectric grid.",
```
Comment (Maleclypse) [inline data/json/furniture_and_terrain/appliances.json:552]:
```suggestion
    "description": "A rather complicated system of thick-walled pipes, powerful compressors, and wiring, meant to interface with a reactor producing ammonia from nitrogen and hydrogen via the Haber-Bosch process, with a power cord to connect it to an electric grid.",
```

**Pull request merged**